python/lsst/daf/persistence/butlerLocation.py

```diff
@@ -193,7 +193,11 @@ class ButlerLocation(yaml.YAMLObject):
     """
 
     yaml_tag = u"!ButlerLocation"
-    yaml_loader = yaml.Loader
+    try:
```

Comment:
Is this actually used anywhere?  It seems strange that this is a class variable rather than built into `from_yaml`, and I don't see any caller for the latter or use of the variable.

Reply:
Weird, I changed it because I assumed it was used but you are right, I can't find anywhere that uses that or the dumper.

Reply:
My mistake. This class inherits from `yaml.YamlObject` and `yaml_loader` is the class variable that `yaml` assumes will define the loader.